moveit_core/robot_trajectory/include/moveit/robot_trajectory/robot_trajectory.h

```diff
@@ -213,7 +220,7 @@ class RobotTrajectory
       to be constructed internally is obtained by copying the reference state and overwriting the content from a
      trajectory point in \e trajectory. */
   void setRobotTrajectoryMsg(const robot_state::RobotState& reference_state,
-                             const trajectory_msgs::JointTrajectory& trajectory);
+                             const trajectory_msgs::msg::JointTrajectory& trajectory);
```

Comment:
same here: do these PRs in bulk. you're not going to find anything surprising by doing this all at once.